tools/enhancements/summary.go

```diff
@@ -100,6 +100,53 @@ func extractSummary(body string) string {
 	return b.String()
 }
 
+// getEnhancementFilename returns the filename of the enhancement
+// document modified by the pull request
+func getEnhancementFilename(pr int) (filename string, err error) {
```

Comment:
not using this anymore, i think?  want to keep it anyway?

Reply:
I went ahead and removed it

Reply:
I also added a Makefile to be able to run the linter locally.